service/web/tabs/app/src/containers/BackfillStatusContainer.tsx

```diff
@@ -392,6 +397,56 @@ class BackfillStatusContainer extends React.Component<
               ))}
             </tbody>
           </HTMLTable>
+
+          <Dialog
+            isOpen={!!this.state.event_data}
+            title={this.state.event_title}
+            onClose={() => this.setState({event_data: null})}
+            style={{width: 1000}}
+          >
+            <div style={{width:'100%', height: 600}}>
+               <AnchorButton text={"Copy"} intent={Intent.PRIMARY} onClick={() => {this.dialogTextRef.current.select(); document.execCommand('copy')}} />
+              <textarea ref={this.dialogTextRef} style={{width: '100%', height: 600}}>
+                 {this.state.event_data}
+              </textarea>
+            </div>
+          </Dialog>
+          <H3>Event log (last 50)</H3>
+          <HTMLTable bordered={true} striped={true} style={{width:"100%"}}>
+            <thead>
+              <tr>
+                <th>Time</th>
+                <th>User</th>
+                <th>Partition</th>
+                <th>Event</th>
+                <th>More data</th>
+              </tr>
+            </thead>
+            <tbody>
+              {status.event_logs.map((event_log: any) => (
+                <tr>
+                  <td style={{paddingTop: 0, paddingBottom: 0}}>{event_log.occurred_at}</td>
```

Comment:
would like to make this local timezone, maybe later

Reply:
Do you know if this defaults to UTC?

Reply:
it does